src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1909,7 +1944,7 @@ public async Task<string> InvokeToStringAsync(IEnumerable<int> typeIds, bool isV
                     foreach (var methodId in methodIds)
                     {
                         var methodInfoFromRuntime = await GetMethodInfo(methodId, token);
-                        if (methodInfoFromRuntime.Info.GetParametersInfo().Length > 0)
+                        if (methodInfoFromRuntime?.Info?.GetParametersInfo()?.Length > 0)
```

Comment:
Should we also check that the return type is `string`?
And this method is `InvokeTostringAsync` is only looking at instance methods, correct?